TestRunner.psm1

```diff
@@ -40,24 +40,43 @@ function Get-ServerMetrics {
     $result
 }
 
-function Duplicate-OdsLogs {
```

Comment:
This function was copying all *.log files into a sub-directory so that it would be easier to bring the sub-directory back to the test host. I've never seen the value in that over the years, as we really only want the current log file, not old log files that were created by log4net when rotating.

Reply:
The default maximumFileSize is 100kB in log4net.config so there is a possibility that file rotates a couple of times during the same run (specially with locust tests) unless the default setting is changed.

Reply:
Hm, might need to capture more then. Whenever I would run the tool in the past, it would just accumulate lots of log files from previous runs, which were not useful.

If we restore this, then I would just have it copy *.log from remote to the local server. I haven't seen any advantage to making temporary copies on the remote server and then copying the temporaries back to the host. In other words, I would try to change the copy command farther down in the file without restoring this particular function.

Reply:
Turns out there is a no-code solution: just use *.log in the `.env` file. I will add that to the user guide. To test that, I have the following line in my `.env` and I created a second log file manually, `test.log`.
```
PERF_API_LOG_FILE_PATH=<shortened>/EdFi.Ods.WebApi/bin/Debug/netcoreapp3.1/*.log
```

In this screenshot you can see that both log files were copied into the test results directory.

![image](https://user-images.githubusercontent.com/9324390/165333434-e7185755-3811-4454-90d7-c282e577f637.png)